patrick/Sean/src/TodoInputField.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+class TodoInputField extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.state = {
+      text: ""
+    };
+  }
+  
+  handleChange = (e) => {
+      this.setState({ text: e.target.value });
+  };
+
+  handleSubmit = (e) => {
+    const text = this.state.text;
+    if (e.which === 13) {
+      this.props.onSave(text);
+      this.setState({ text: "" });
+    }
+  };
+  
+  render() {
+    return (
+      <input 
+        type="text" 
+        placeholder="Enter a name" 
+        value={this.state.text}
+        onChange={this.handleChange}
+        onKeyDown={this.handleSubmit}
```

Comment:
This will work, however I like to set my `Submit` handlers to a form or a button. For this repo and sprint using it with a keyDown works if you're checking for the "enter" key.. There are more "natural" ways to accomplish this like putting the input field inside a `form onSubmit={submitHandler}`